array-manipulation-3/src/flatten.js

```diff
@@ -0,0 +1,13 @@
+/* exported flatten */
+function flatten(array) {
+  var output = [];
+  for (var i = 0; i < array.length; i++) {
+    for (var j = 0; j < array[i].length; j++) {
+      output.push(array[i][j])
+    }
+    if (typeof array[i] === 'number' || !array[i]) {
```

Comment:
It's not safe to assume that your inputs will always be numbers, try writing your solution so that it works with any data type in the array.